docs/tutorials/04-include-in-your-build-tool.md

```diff
@@ -13,14 +13,28 @@ In this tutorial I'll show you how to create a simple CLI tool that creates an i
 
 Bootstrap the installation of development dependencies of Buildah by following the [Building from scratch](https://github.com/slinkydeveloper/buildah/blob/master/install.md#building-from-scratch) instructions and in particular creating a directory for the Buildah project by completing the instructions in the [Installation from GitHub](https://github.com/containers/buildah/blob/master/install.md#installation-from-github) section of that page.
 
-Now let's bootstrap our project. Assuming you are in the directory of the project, run:
+Now let's bootstrap our project. Assuming you are in the directory of the project, run the following to initialize the go modules:
 
 
 ```shell
 go mod init
 ```
 
-To initialize the go modules. Now import Buildah as a dependency:
+Next, we should import Buildah as a dependency. However, make sure that you have the following
+developer packages installed:
+
+```shell
+dnf install btrfs-progs-devel gpgme-devel device-mapper-devel
```

Comment:
Some of these can be avoided via build tags.

Buildah is currently built with
Buildtags = "containers_image_openpgp exclude_graphdriver_devicemapper"

Reply:
I think you can get rid of btrfs with
CGO_ENABLED=0  go build ...

Reply:
Well, some might actually use Btrfs. When installing OpenSUSE, Btrfs is the first option in the filesystem drop-down, so someone might even install it by mistake :). Putting a matrix of all possible installation / build scenarios would unnecessarily clutter the tutorial. IMHO, the tutorial should be doable without ugly error messages in the first place. Optimization options could be explored later.

Reply:
Almost eveyone in the world uses overlay, so while podman can work with btrfs storage backend, it is not well tested.  Podman using overlayfs works fine, The compilation errors are about support for btrfs and devicemapper.
Devicemapper is NOT a supported back end for podman or buildah.